Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4168,3 +4170,39 @@ bool CLuaVehicleDefs::BlowVehicle(CClientEntity* entity, std::optional<bool> wit
 {
     return CStaticFunctionDefinitions::BlowVehicle(*entity, withExplosion);
 }
+
+std::variant<bool, std::array<std::array<float, 3>, 4>> CLuaVehicleDefs::GetVehicleEntryPoints(CClientVehicle* vehicle)
+{
+    if (CClientVehicleManager::GetMaxPassengerCount(vehicle->GetModel()) == 255)
+    {
+        return false;
+    }
+
+    std::array<std::array<float, 3>, 4> entryPoints;
+
+    for (unsigned int i = 0; i < 4; i++)
+    {
+        CVector entryPoint = vehicle->GetEntryPoint(i);
+
+        entryPoints[i] = {entryPoint.fX, entryPoint.fY, entryPoint.fZ};
+    }
+
+    return entryPoints;
+}
+
+std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::OOP_GetVehicleEntryPoints(CClientVehicle* vehicle)
+{
+    if (CClientVehicleManager::GetMaxPassengerCount(vehicle->GetModel()) == 255)
+    {
+        return false;
+    }
+
+    std::array<CVector, 4> entryPoints;
+
+    for (unsigned int i = 0; i < 4; i++)
+    {
+        entryPoints[i] = vehicle->GetEntryPoint(i);
+    }
+
+    return entryPoints;
```

Comment:
The function is the same as `GetVehicleEntryPoints`.
Link it with that function instead of creating a new one with the same definition